### ISSUE TITLE
Alert Rule Policy Filter

### ISSUE THREAD

**Issue by lvets** [PaloAltoNetworks/terraform-provider-prismacloud#277]
## Describe the bug

It's unclear from the documentation how to use the `alertRulePolicyFilter` part in the Terraform alert rule resource.

Let's say the API returns the following (removed some fields from the snippet for clearity):

```json
{
  "policyScanConfigId": "xxx",
  "name": "Prisma Default Alert Rule",
  "description": "Prisma Default alert rule to scan all policies associated with label Prisma_Cloud  and all account added to 'Default Account Group'",
...
  "target": {
    "accountGroups": [
      "xxx"
    ],
    "excludedAccounts": [],
    "regions": [],
    "tags": [],
    "alertRulePolicyFilter": {
      "policy.severity": [],
      "policy.label": [
        "Prisma_Cloud"
      ],
      "cloud.type": [],
      "policy.complianceStandard": []
    }
  },
...
}
```

How can I use the `alertRulePolicyFilter` parameters in the Terraform `prismacloud_alert_rule` resource:

```json
    "alertRulePolicyFilter": {
      "policy.severity": [],
      "policy.label": [
        "Prisma_Cloud"
      ],
      "cloud.type": [],
      "policy.complianceStandard": []
    }
```

Or do I need to first filter the `prismacloud_policies` data resource and then use those policies in a `policies` argument in the `prismacloud_alert_rule` resource?

**Comment by bryonwinger** [comment]
Attention to this would be great. We really need this fixed for our use cases.

**Comment by AnushreeHS** [comment]
Hi @bryonwinger @lvets This feature is implemented in the latest release (v1.5.7).
